.github/workflows/deploy.yml

```diff
@@ -0,0 +1,21 @@
+name: DR2 Release Binary
```

Comment:
Should we add a more descriptive name, like `DR2 Release Package Anonymiser Binary`?

Reply:
I'm not sure about "Package". DR2 Release Anonymiser Binary sounds ok though.